tutorials/hana-cloud-dl-clients-overview/hana-cloud-dl-clients-overview.md

```diff
@@ -394,7 +415,7 @@ The data lake client install includes [Interactive SQL Client (DBISQL)](https://
     >DBISQL can also be started without a GUI.
     >
     >```Shell (Windows)
-    dbisql -c "uid=USER1;pwd=Password1;host=XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX.iq.hdl.trial-XXXX.hanacloud.ondemand.com:443;ENC=TLS(tls_type=rsa;direct=yes)" -nogui
+    dbisql -c "uid=USER1;pwd=Password1;host=XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX.iq.hdl.xxxx-xxxx.hanacloud.ondemand.com:443;ENC=TLS(tls_type=rsa;direct=yes)" -nogui
     >```
     >
     > Note that in a Bash shell, strings in double quotes versus single quotes are treated [differently](https://stackoverflow.com/questions/6697753/difference-between-single-and-double-quotes-in-bash).

```

Comment:
Please rephrase and do not start a note with the keyword "Note", but use a label instead, for example, \*\*Important:\*\*